README.md

```diff
@@ -76,8 +76,9 @@ USER root
 RUN apt-get update && apt-get -y install libv8-dev
 
 COPY . /home/rstudio/
+RUN chown -R rstudio:rstudio * .*
```

Comment:
In the docs this is 
```
COPY . ${HOME}
RUN chown -R ${NB_USER} ${HOME}
```
But I wrote it to be explicit that the user was rstudio because it's more intelligible to my users, and I thought the user always has to be rstudio for this to work...?

Reply:
RStudio doesn't care what the user name is; you simply need a valid linux user with a password who has write permission to their own $HOME directory.  The upstream Docker container has only created the user `rstudio` though, so if you want a different name, you have to create a new user (e.g. `useradd ...`, though that can create issues when working with linked volumes if user isn't careful to match UIDs. ).  

All that is a essentially a technical tangent though -- upshot is either option will work just fine (because `rocker/binder` image configures the above env vars  so `NB_USER=rstudio` anyway).  So just put whatever liiks less confusing to a user.